recipes/boost/meta.yaml

```diff
@@ -0,0 +1,98 @@
+  name: boost
+  version: {{ version }}
+
+source:
+  fn:  {{ filename }}
+  url: http://sourceforge.net/projects/boost/files/boost/{{ version }}/{{ filename }}
+  md5: 65a840e1a0b13a558ff19eeb2c4f0cbe
+
+build:
+  features:
+    - vc9               # [win and py27]
+    - vc10              # [win and py34]
+    - vc14              # [win and py35]
+
+requirements:
+  build:
+    - gcc               # [linux]
+    - python
+    - icu               # [unix]
+    - bzip2             # [unix]
+    - zlib
+
+  run:
+    - libgcc            # [linux]
+    - python
+    - icu               # [unix]
+    - zlib
+
+test:
+  commands:
+    # Verify static libraries.
+    - test -f $PREFIX/lib/libboost_chrono.a                   # [unix]
+    - test -f $PREFIX/lib/libboost_exception.a                # [unix]
+    - test -f $PREFIX/lib/libboost_system.a                   # [unix]
+    - test -f $PREFIX/lib/libboost_test_exec_monitor.a        # [unix]
+    - test -f $PREFIX/lib/libboost_timer.a                    # [unix]
+
+    # Verify dynamic libraries.
+    {% set win_vstr = "_".join(version.split(".")[:2]) %}
+    {% set boost_libs = [
+            "atomic",
+            "chrono",
+            "container",
+            "context",
+            "coroutine",
+            "date_time",
+            "filesystem",
+            "graph",
+            "iostreams",
+            "locale",
+            "log",
+            "log_setup",
+            "math_c99",
+            "math_c99f",
+            "math_c99l",
+            "math_tr1",
+            "math_tr1f",
+            "math_tr1l",
+            "prg_exec_monitor",
+            "program_options",
+            "python",
+            "random",
+            "regex",
+            "serialization",
+            "signals",
+            "system",
+            "thread",
+            "timer",
+            "type_erasure",
+            "unit_test_framework",
+            "wave",
+            "wserialization"
+    ] %}
+    {% for each_boost_lib in boost_libs %}
+    - test -f $PREFIX/lib/libboost_{{ each_boost_lib }}.dylib                                             # [osx]
+    - test -f $PREFIX/lib/libboost_{{ each_boost_lib }}.so                                                # [linux]
+    - if not exist %PREFIX%\\Library\\bin\\boost_{{ each_boost_lib }}-vc90-mt-{{ win_vstr }}.dll exit 1   # [win and py27]
+    - if not exist %PREFIX%\\Library\\bin\\boost_{{ each_boost_lib }}-vc100-mt-{{ win_vstr }}.dll exit 1  # [win and py34]
+    - if not exist %PREFIX%\\Library\\bin\\boost_{{ each_boost_lib }}-vc140-mt-{{ win_vstr }}.dll exit 1  # [win and py35]
+    - if not exist %PREFIX%\\Library\\lib\\boost_{{ each_boost_lib }}-vc90-mt-{{ win_vstr }}.lib exit 1   # [win and py27]
+    - if not exist %PREFIX%\\Library\\lib\\boost_{{ each_boost_lib }}-vc100-mt-{{ win_vstr }}.lib exit 1  # [win and py34]
+    - if not exist %PREFIX%\\Library\\lib\\boost_{{ each_boost_lib }}-vc140-mt-{{ win_vstr }}.lib exit 1  # [win and py35]
```

Comment:
Can someone with more Windows knowledge clarify as to whether this looks correct?